pkg/libvirt/constants.go

```diff
@@ -1,8 +1,9 @@
 package libvirt
 
+var DriverVersion = "dev"
```

Comment:
Won't this make it harder to test the machine driver from git? What I'm doing at the moment is `make local`, `cp crc-driver-libvirt-local ~/.crc/machines/crc`, adjust the expected version number in crc if needed. Not clear how crc version check will react to a `dev` version? You mention doing `gorelease --snapshot`, does this mean one will *have to* use this instead of `make local` after this change?
Imo it would be useful to detail in the commit log what `make release` is going to do with `goreleaser`, what it gives us that we don't have now, ...

Reply:
I added the version in the Makefile and expanded the commit log. 

I don't like bump commit. They are noisy and we loose time with that.
Now you can do `VERSION=0.12.12 make local` and use goreleaser to find the correct version automatically.

Reply:
I'm actually quite confused by the new log :) 

> How-to release: create the tag and just run goreleaser.

Shouldn't this be `make release` rather than `goreleaser`? How are the Makefile variable `VERSION?=dev` and goreleaser `{Version}` interacting with git tags? Regarding the uploaded binary, it's built locally on the machine where `make release` was run, and then uploaded? Or is there something more sophisticated going on? Could it use `make` rather than duplicating "how to build" knowledge in a 4th place?

Reply:
> Shouldn't this be make release rather than goreleaser?

Yes, changed.

> How are the Makefile variable VERSION?=dev and goreleaser {Version} interacting with git tags? 

VERSION is ignored. goreleaser only looks at the git tag.

> Regarding the uploaded binary, it's built locally on the machine where make release was run, and then uploaded? 

Yes.

> Could it use make rather than duplicating "how to build" knowledge in a 4th place?

Apparently no :( We can hack it by using hooks but I would rather not to do that.

Reply:
Actually I don't think it's a good move to start doing builds on the machine of the person releasing, and not even using a build container. If we build using the centos container, and upload this, this would give at least a similar workflow to what we are currently doing, and we can revisit this once we build differently if needed.

Reply:
Yes you're right. We should run goreleaser in the CI, in a separated job. But we still need this kind of tool.

Reply:
If we can release as part of a buid pipeline, yes.
even if performed from public CI this would be an improvement over the current practice.

however, it is more likely we will look into brew for dowwnstream

Reply:
The next step is to use this kind of pipeline https://github.com/goodhosts/cli/blob/master/.github/workflows/release.yml
We will have a fully automated release in GitHub.

The other option is to use digitalocean library and compile this driver directly in crc.